modules/nw-sriov-networknodepolicy-object.adoc

```diff
@@ -27,62 +27,56 @@ spec:
 
-<12> An array of one or more PCI bus addresses for the PF of the device. Provide the address in the following format: `0000:02:00.1`.
+<12> Optional: An array of one or more PCI bus addresses for the PF of the device. Provide the address in the following format: `0000:02:00.1`.
 
-<13> Optional: The driver type for the virtual functions. The only allowed values are `netdevice` and `vfio-pci`. The default value is `netdevice`.
+<13> Optional: The platform-specific network filter. The only supported platform is {rh-openstack-first}. Acceptable values use the following format: `openstack/NetworkID:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`. Replace `xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx` with the value from the `/var/config/openstack/latest/network_data.json` metadata file.
+
+<14> Optional: The driver type for the virtual functions. The only allowed values are `netdevice` and `vfio-pci`. The default value is `netdevice`.
 +
-[NOTE]
-====
-For a Mellanox card to work in Data Plane Development Kit (DPDK) mode on bare metal nodes, use the `netdevice` driver type and set `isRdma` to `true`.
-====
+For a Mellanox NIC to work in Data Plane Development Kit (DPDK) mode on bare metal nodes, use the `netdevice` driver type and set `isRdma` to `true`.
 
-<14> Optional: Whether to enable remote direct memory access (RDMA) mode. The default value is `false`.
+<15> Optional: Whether to enable remote direct memory access (RDMA) mode. The default value is `false`.
 +
-[NOTE]
-====
-If the `isRDMA` parameter is set to `true`, you can continue to use the RDMA enabled VF as a normal network device.
-A device can be used in either mode.
-====
+If the `isRDMA` parameter is set to `true`, you can continue to use the RDMA-enabled VF as a normal network device. A device can be used in either mode.
 
-<15> Optional: The link type for the VFs. You can specify one of the following values: `eth` or `ib`. `eth` is ethernet and `ib` is InfiniBand. The default value is `eth` if it is not explicitly set. When `linkType` is set to `ib`, `isRdma` will be set to `true` by SR-IOV Network Operator webhook automatically. When `linkType` is set to `ib`, `deviceType` should not be set to `vfio-pci`.
+<16> Optional: The link type for the VFs. You can specify one of the following values: `eth` or `ib`. Specify `eth` for Ethernet or `ib` for InfiniBand. The default value is `eth`.
++
+When `linkType` is set to `ib`, `isRdma` is automatically set to `true` by an SR-IOV Network Operator webhook. When `linkType` is set to `ib`, `deviceType` should not be set to `vfio-pci`.
 
 [id="sr-iov-network-node-configuration-examples_{context}"]
 == SR-IOV network node configuration examples
 
-The following example describes the configuration for an IB device:
+The following example describes the configuration for an InfiniBand device:
 
-.Example configuration for an IB device
+.Example configuration for an IniniBand device
```

Comment:
nit: IniniBand -> InfiniBand.

Reply:
Ick.  Getting the spellchecker for tech terms is on the todo list.  Thank you.